src/reducers/ui.js

```diff
@@ -1,15 +1,19 @@
 import Immutable from 'immutable';
 import pick from 'lodash/pick';
-import {updateVerticalFlex} from '../util/resize';
+import {updateFlex} from '../util/resize';
 
-export const DEFAULT_VERTICAL_FLEX = new Immutable.List(['1', '1', '1']);
+const DEFAULT_FLEX = new Immutable.List(['1', '1', '1']);
+export const DEFAULT_WORKSPACE_FLEX = new Immutable.Map({
+  columnFlex: DEFAULT_FLEX,
```

Comment:
(And with that in mind, I’d probably rename `DEFAULT_WORKSPACE_FLEX` to `DEFAULT_FLEX` and declare `DEFAULT_ROW_FLEX` and `DEFAULT_COLUMN_FLEX`)